ddp-workspace/projects/ddp-dsm-ui/src/app/utils/utils.ts

```diff
@@ -96,7 +96,11 @@ export class Utils {
     let csv = this.makeCSV(data, paths, columns);
     csv = headers + '\r\n' + csv;
     const blob = new Blob([ csv ], {type: 'text/csv;charset=utf-8;'});
+    /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
+    // @ts-ignore
```

Comment:
why do we need it?

Reply:
it is DSM legacy stuff.

Reply:
since Microsoft stopped supporting IE oficcially this year we probably need to clean up such code. I created the tech debt task for that 
https://broadinstitute.atlassian.net/browse/DDP-7420